dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/FileUtilsTest.java

```diff
@@ -143,4 +143,8 @@ public void tearDown() {
         FileUtils.deleteFile("test");
     }
 
+    @Test
+    void createFileWith755() throws IOException {
+        FileUtils.createFileWith755(Paths.get("/Users/wenjun/a/b/test.txt"));
```

Comment:
Perhaps a temporary directory would be more appropriate

Reply:
Yes, this change should be removed, this is my test code.